extensions/vtk/package.json

```diff
@@ -56,7 +56,6 @@
     "@ohif/ui": "^0.58.4",
     "cornerstone-tools": "^4.0.9",
     "cornerstone-wado-image-loader": "^3.0.0",
-    "dcmjs": "^0.6.1",
```

Comment:
@JamesAPetts can you confirm if this line is really necessary? I have removed, but I can come back in case its needed.

Reply:
Yeah that doesn't need to be in dev dependencies 👍 .